src/components/Dialog/Newconnections.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <Dialog
+    class="connection-dialog"
+    :visible="true"
+    modal
+    :closable="true"
+    :breakpoints="{ '960px': '75vw', '640px': '100vw' }"
+  >
+    <template #header>
+      <span class="p-dialog-title"> New Connections </span>
```

Comment:
Internationalization is missing in the component.

Reply:
Added Internationalization